pyproject.toml

```diff
@@ -62,6 +62,11 @@ docs = ["mkdocs", "mkdocstrings", "mkdocs-material", "mkdocs-autorefs", "psycopg
 
 [tool.isort]
 profile = "black"
+[tool.black]
+line-length = 88
+target-version = ["py39"]
```

Comment:
same here, lowest python version we support

Reply:
Should match the version on tox and github action. Pre-commit is only run for `py39`.